File: app/src/main/java/com/nerdcutlet/skylarktest/di/setdetail/SetDetailModule.java
package com.nerdcutlet.skylarktest.di.setdetail;

import com.nerdcutlet.skylarktest.ui.setdetail.SetDetailActivity;

import dagger.Module;

/**
 * Created by Aldrich on 16-Nov-17.
 */
@Module
public class SetDetailModule {

    private final SetDetailActivity setDetailActivity;

    public SetDetailModule(SetDetailActivity setDetailActivity) {
        this.setDetailActivity = setDetailActivity;
    }
}

File: app/src/main/java/com/nerdcutlet/skylarktest/di/setdetail/SetDetailModule.kt
package com.nerdcutlet.skylarktest.di.setdetail;

import com.nerdcutlet.skylarktest.ui.setdetail.SetDetailActivity;

import dagger.Module;
import dagger.Provides

/**
 * Created by Aldrich on 16-Nov-17.
 */
@Module
class SetDetailModule(private val setDetailActivity: SetDetailActivity) {

    @Provides
    fun provideSetDetailActivity(): SetDetailActivity = setDetailActivity

}
